app/assets/javascripts/discourse/app/form-kit/components/fk/control/select.gjs

```diff
@@ -15,12 +15,17 @@ const SelectOption = <template>
 export default class FKControlSelect extends Component {
   static controlType = "select";
 
+  get includeNone() {
+    return this.args.field.validation !== "required";
```

Comment:
I wonder if we you can't do: `this.args.field.required` which is probably better

Reply:
Regardless of what the validation is set to, `required` has this object `{ trim: false }` as its value.

Reply:
hmmm... ok you can merge like this, I will dig that later :+1: thanks